oak_functions/examples/weather_lookup/scripts/cloud_run_deploy

```diff
@@ -12,13 +12,43 @@ export EXAMPLE_NAME='weather_lookup'
 export FUNCTIONS_INSTANCE_NAME='oak-functions-weather-lookup'
 export CLOUD_RUN_SERVICE_NAME='weather-lookup-endpoint'
 export API_TITLE='Oak Functions Weather Lookup Endpoint'
-
+export CONNECTOR_NAME="weather-lookup-connector"
+
```

Comment:
Super nit:  in the interest of consistency, a single blank line as a separator would be preferred.